Kernel/Heap/Heap.h

```diff
@@ -20,6 +20,10 @@ class Heap {
 
     struct AllocationHeader {
         size_t allocation_size_in_chunks;
+#if ARCH(X86_64)
+        // FIXME: Get rid of this somehow
+        size_t alignment_dummy;
```

Comment:
maybe an alignas(void*) or does this do the wrong offset

Reply:
Both works, this felt slightly better, but I'm fine with either.